Libraries/DirectMLX.h

```diff
@@ -2875,6 +2875,107 @@ namespace dml
 
 #endif // DML_TARGET_VERSION >= 0x3100
 
+#if DML_TARGET_VERSION >= 0x4100
```

Comment:
The order of these functions should match the order that they're defined in DirectML.h, so these changes should be moved down to the bottom of the file alongside RoiAlignGrad.

Reply:
The current convention appears to be grouping the functions according to operator similarity